modules/RTC/JitsiLocalTrack.js

```diff
@@ -392,6 +391,22 @@ export default class JitsiLocalTrack extends JitsiTrack {
 
         this._setEffectInProgress = true;
 
+        // For firefox/safari, replace the stream without doing a offer answer with the remote peer.
```

Comment:
Please don't mention browser names since that's bound to change.

Reply:
ok